src/domains/project/service/project.service.impl.ts

```diff
@@ -13,6 +14,43 @@ export class ProjectServiceImpl implements ProjectService {
     private readonly userRepository: UserRepository
   ) {}
 
+  /**
+   * Fetches detailed information about a specific project.
+   *
+   * @param {GetProjectParams} input - An object containing user ID and project ID for lookup.
+   * @param {string} input.userId - ID of the user requesting project information.
+   * @param {string} input.projectId - ID of the project to retrieve details for.
+   * @returns {Promise<MyProjectViewDTO>} - Promise resolving to a ProjectViewDTO object with project details.
+   * @throws {NotFoundException} - Throws NotFoundException with specific messages:
+   *                                 * 'User': If the user with the provided ID is not found.
+   *                                 * 'Project': If the project with the provided ID is not found or the user doesn't have access to it.
+   */
+  async getMyProjectView({ userId, projectId }: GetProjectParams): Promise<MyProjectViewDTO> {
+    const user = await this.userRepository.getByCognitoId(userId);
+    if (user == null) throw new NotFoundException('User');
+
+    const project: ProjectDetailsDTO | null = await this.projectRepository.getProjectDetails(userId, projectId);
+    if (project == null) throw new NotFoundException('Project');
+
+    const pmUserAndRole: UserRoleDTO | undefined = project.usersAndRoles.find((userAndRole) => userAndRole.role.name === ProjectRole.PROJECT_MANAGER);
```

Comment:
check this method, is not working ok

Reply:
![image](https://github.com/sirius-valley/tricker-backend/assets/70229654/61f8d757-e837-48cc-a26c-3687c1f8ba26)

working now